changelog.d/14181.bugfix

```diff
@@ -0,0 +1 @@
+Fix poor performance of the `event_push_backfill_thread_id` background update, which was introduced in Synapse 1.68.0rc1.
```

Comment:
Which PR is this a follow-up to? (add to description)

Reply:
I cross-linked to #13753